org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/trace/TraceEditor.java

```diff
@@ -109,6 +119,68 @@ public void selectionChanged(SelectionChangedEvent event) {
 				}
 			}
 		});
+		Menu contextMenu = new Menu(text);
+		MenuItem copyItem = new MenuItem(contextMenu, SWT.PUSH);
+		copyItem.setText("&Copy");
+		copyItem.setAccelerator('C');
```

Comment:
![Bildschirmfoto 2021-02-19 um 09 45 12](https://user-images.githubusercontent.com/238736/108480509-5a90cf80-7297-11eb-925f-ecf310a72dbd.png)
![Bildschirmfoto 2021-02-19 um 09 43 37](https://user-images.githubusercontent.com/238736/108480511-5b296600-7297-11eb-9a4a-a27b4ab876da.png)
there is something wrong about the shortcut

Reply:
Fixed on MacOS. Modifier ist not shown on Windows/Linux. Probably an issue with with eclipse shortcut system. But hotkey works, just a minor display issue.